File: app/src/main/java/com/appchamp/wordchunks/ui/game/CustomGridLayoutManager.java
package com.appchamp.wordchunks.ui.game;

import android.content.Context;
import android.support.v7.widget.GridLayoutManager;

/*
 * Disables scrolling in grids.
 */
public class CustomGridLayoutManager extends GridLayoutManager {

    public CustomGridLayoutManager(Context context, int spanCount) {
        super(context, spanCount);
    }

    @Override
    public boolean canScrollVertically() {
        return false;
    }
}

File: app/src/main/java/com/appchamp/wordchunks/ui/game/CustomGridLayoutManager.kt
package com.appchamp.wordchunks.ui.game

import android.content.Context
import android.support.v7.widget.GridLayoutManager

/*
 * Disables scrolling in grids.
 */
class CustomGridLayoutManager(context: Context, spanCount: Int) :
        GridLayoutManager(context, spanCount) {

    override fun canScrollVertically(): Boolean {
        return false
    }
}
